### PR TITLE
add romanian language

### PR DESCRIPTION
**Pull request by paultimoce**

### CODE REVIEW

**Review by jaanonim — APPROVED** [review]

**Comment by jaanonim** [comment]
@paultimoce please make shore that key `ro.json` of dict are the same as in `books.json` file.

File: data/books/ro.json [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        "1imp",
+        "1împ"
+    ],
+    "2IMP": [
+        "2împărați",
+        "2imparati",
+        "2imp",
+        "2împ"
+    ],
+    "1CRON": [
```
Comment (jaanonim) [inline data/books/ro.json:54]: Example of invalid key

**Comment by jaanonim** [comment]
When you fix it open new PR. 
(Also it would be nice to write something in descripton of a PR when you open one on sb repo)

File: data/books/ro.json (hunk 1 above)
Comment (paultimoce) [reply data/books/ro.json:54]: indeed, I went out of my way yesterday to change the keys also to romanian, I now see they should be left in english

**Pull request merged**